cborg/src/Codec/CBOR/Magic.hs

```diff
@@ -120,13 +120,19 @@ import qualified Numeric.Half as Half
 import           Data.Bits ((.|.), unsafeShiftL)
 #endif
 
-#if defined(ARCH_32bit)
-import           GHC.IntWord64 (wordToWord64#, word64ToWord#,
-                                intToInt64#, int64ToInt#,
-                                leWord64#, ltWord64#, word64ToInt64#)
-
+#if MIN_VERSION_base(4,17,0)
+import           GHC.Exts as Compat (wordToWord64#, word64ToWord#,
+                                     intToInt64#, int64ToInt#,
+                                     leWord64#, ltWord64#, word64ToInt64#)
+                                
+#else
+import           GHC.IntWord64 as Compat (wordToWord64#, word64ToWord#,
```

Comment:
GHC.IntWord64, doesn't exist in base-4.16.3.0, so this is currently failing for GHC 9.2.4. Other alternatives need to be found for this version. GHC.IntWord64 is removed in ghc-prim-0.8.0